src/main/java/com/harrison/controllers/SuggestedSkillsController.java

```diff
@@ -1,27 +1,51 @@
 package com.harrison.controllers;
 
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
 import java.util.List;
 
+import javax.sql.DataSource;
+
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.harrison.enums.Proficiency;
 import com.harrison.suggestedskills.domain.SuggestedSkill;
-import com.harrison.suggestedskills.repository.SuggestedSkillRepository;
 
 @RestController
 @RequestMapping(value = "/api/suggested/skills")
 public class SuggestedSkillsController {
 
     @Autowired
-    private SuggestedSkillRepository suggestedSkillRepository;
+    @Qualifier("suggestedSkillsDataSource")
+    private DataSource suggestedSkillsDataSource;
 
     @RequestMapping(method = RequestMethod.GET)
-    public ResponseEntity<List<SuggestedSkill>> getSkills() {
-        return new ResponseEntity<>(suggestedSkillRepository.findAll(), HttpStatus.OK);
+    public ResponseEntity<List<SuggestedSkill>> getSkills() throws SQLException {
+        String sql = "select * from suggested_skill";
+        ResultSet rs;
+        Connection conn = suggestedSkillsDataSource.getConnection();
+        try {
+            rs = conn.prepareStatement(sql).executeQuery();
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Close this "PreparedStatement". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS2095)